app/javascript/utils/VaccineAdvancedFilters.js

```diff
@@ -0,0 +1,55 @@
+import _ from 'lodash';
```

Comment:
Commenting more as a mental note to myself that this file won't be necessary with my refactor